hack/yaml-lint.sh

```diff
@@ -1,11 +0,0 @@
-#!/bin/sh
```

Comment:
It didn't look like we actually ran it anywhere? Like if it actually worked/ran/passed that seems nice to have so we know if we goofed, but I didn't see CI call it, and it doesn't seem like it was wired up to anything. Did I miss it?

Reply:
No I don't think it was running in CI either :confused:. What do you think we should do? I'm with just skipping it.

Reply:
hahaha well the last time we fixed anything it caught was 3 years ago according to blame, and we're currently naughty in the main branch. 

And since we're copying those vpa manifests from `kuberenetes-autoscaler`, messing with the yaml here is just going to ruin the diff. But that still leaves everything we don't copy. 

How about...we leave it out and make a JIRA card to tweak the rules and re-add yaml-lint.sh and a makefile target someday? 
```
install/deploy/01_vpacontroller.crd.yaml
  27:121    warning  line too long (171 > 120 characters)  (line-length)
  32:121    warning  line too long (175 > 120 characters)  (line-length)
  91:121    warning  line too long (156 > 120 characters)  (line-length)
  94:121    warning  line too long (154 > 120 characters)  (line-length)
  101:121   warning  line too long (156 > 120 characters)  (line-length)
  161:121   warning  line too long (156 > 120 characters)  (line-length)
  164:121   warning  line too long (154 > 120 characters)  (line-length)
  171:121   warning  line too long (156 > 120 characters)  (line-length)
  231:121   warning  line too long (156 > 120 characters)  (line-length)
  234:121   warning  line too long (154 > 120 characters)  (line-length)
  241:121   warning  line too long (156 > 120 characters)  (line-length)

install/deploy/05_vpa-crd.yaml
  121:121   warning  line too long (150 > 120 characters)  (line-length)
  132:121   warning  line too long (150 > 120 characters)  (line-length)
  292:121   warning  line too long (150 > 120 characters)  (line-length)
  305:121   warning  line too long (150 > 120 characters)  (line-length)
  314:121   warning  line too long (150 > 120 characters)  (line-length)
  330:121   warning  line too long (150 > 120 characters)  (line-length)
  401:121   warning  line too long (150 > 120 characters)  (line-length)
  412:121   warning  line too long (150 > 120 characters)  (line-length)
  532:121   warning  line too long (150 > 120 characters)  (line-length)
  545:121   warning  line too long (150 > 120 characters)  (line-length)
  554:121   warning  line too long (150 > 120 characters)  (line-length)
  570:121   warning  line too long (150 > 120 characters)  (line-length)

manifests/stable/vertical-pod-autoscaler-controller.crd.yaml
  27:121    warning  line too long (171 > 120 characters)  (line-length)
  32:121    warning  line too long (175 > 120 characters)  (line-length)
  91:121    warning  line too long (156 > 120 characters)  (line-length)
  94:121    warning  line too long (154 > 120 characters)  (line-length)
  101:121   warning  line too long (156 > 120 characters)  (line-length)
  161:121   warning  line too long (156 > 120 characters)  (line-length)
  164:121   warning  line too long (154 > 120 characters)  (line-length)
  171:121   warning  line too long (156 > 120 characters)  (line-length)
  231:121   warning  line too long (156 > 120 characters)  (line-length)
  234:121   warning  line too long (154 > 120 characters)  (line-length)
  241:121   warning  line too long (156 > 120 characters)  (line-length)

manifests/stable/vpa-v1.crd.yaml
  121:121   warning  line too long (150 > 120 characters)  (line-length)
  132:121   warning  line too long (150 > 120 characters)  (line-length)
  292:121   warning  line too long (150 > 120 characters)  (line-length)
  305:121   warning  line too long (150 > 120 characters)  (line-length)
  314:121   warning  line too long (150 > 120 characters)  (line-length)
  330:121   warning  line too long (150 > 120 characters)  (line-length)
  401:121   warning  line too long (150 > 120 characters)  (line-length)
  412:121   warning  line too long (150 > 120 characters)  (line-length)
  532:121   warning  line too long (150 > 120 characters)  (line-length)
  545:121   warning  line too long (150 > 120 characters)  (line-length)
  554:121   warning  line too long (150 > 120 characters)  (line-length)
  570:121   warning  line too long (150 > 120 characters)  (line-length)
```

Reply:
Sure sounds good :) I can put in a ticket later.